public/index.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!--Import Fira Sans Light 300 & Medium 500 Font-->
```

Comment:
If we're putting it in the html, cleaner to use a `link` element; 

``
 <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300;500&display=swap" rel="stylesheet"> 
```

Also, maybe put it next to another `link` element for better organization?

Reply:
Thanks for the advice, just commited a change with a href link instead of import.